### DISCUSSION TITLE
[False Positive] Specifically the trojan is wacatac, if anyone was curious

### DISCUSSION THREAD

**Discussion by Vivian-Green** [KrisCris/Palworld-Pal-Editor#44] [The Cringed City]
lmao edit if straight L

**Comment by KrisCris** [comment]
False positive, if anyone interested

**Comment by Vivian-Green** [comment]
your false positive disabled multiple security settings in seconds

**Comment by Vivian-Green** [comment]
at which point my ethernet cable was immediately unplugged so unless it took you less than a second to grab what you needed from my craptop, L

**Comment by KrisCris** [comment]
If you have any knowledge in computer security, and aware that pyinstaller or similar tools work by self-extracting, then you know why it's flagged virus sometimes. If you are not happy with the executable, just run the code yourself. LMAO

And if you understand what "verified internally" means on nexusmods, you will not open this wonderful issue.
